ecosystem/platform/server/db/migrate/20220713205849_create_proposals.rb

```diff
@@ -0,0 +1,8 @@
+class CreateProposals < ActiveRecord::Migration[7.0]
+  def change
+    create_table :proposals do |t|
+
+      t.timestamps
```

Comment:
I assume there's more to be added here? Any reason to review/check this in before then?